projects/igniteui-angular/src/lib/core/styles/themes/schemas/dark/_grid.scss

```diff
@@ -74,13 +78,28 @@ $_base-dark-grid: (
         hexrgba: #222
     ),
 
+    cell-selected-within-background: (
+        igx-color: ('secondary', 500),
+        rgba: .24,
+        hexrgba: #222,
+        lighten: 8%
```

Comment:
I really hate that we have to do all this :( Isn't there a color from the secondary palette that is close to the color produced from all this?

Reply:
This is the closest I can get  [screenshot 1](https://i.imgur.com/iBuWFwx.png), [screenshot 2](https://i.imgur.com/w3ey2DX.png)

Reply:
The results in screenshot 1 look decent.